ci/conda_requirements.txt

```diff
@@ -4,7 +4,7 @@ pycodestyle
 flake8
 IPython>4.0.0
 notebook
-scikit-bio=0.5.1
+scikit-bio=0.5.5
```

Comment:
Maybe this should be a minimum pin, instead of an absolute pin, like what you did below in `setup.py`?

Reply:
nice catch! Yes, that should have been a min pin